arbitrum-docs/how-arbitrum-works/timeboost/gentle-introduction.md

```diff
@@ -0,0 +1,121 @@
+#### Timeboost unlocks a new value accrual path for chain owners
+- Chain owners may use Timeboost to capture a portion of the available MEV on their chain that would have otherwise went entirely to searchers. There are many flavors of this too - including the use of custom gas tokens and/or redistribution of these proceeds back to the applications and users on the chain.
+#### Timeboost may help reduce spam and congestion on a network
+- By introducing the ability to “purchase a time-advantage” through the Timeboost auction, it is expected that rational, profit-seeking actors will spend on auctions _instead of_ investing on hardware or infrastructure to win latency races. This diversion of resources by these actors is expected to reduce FCFS MEV-driven spam on the network. 
+
+## What is Timeboost and how does it work?
+
+Timeboost is a _transaction ordering policy_. It's a set of rules that the sequencer of an Arbitrum chain is trusted to follow when ordering transactions that are submitted by users. In the near future, multiple sequencers will be able to enforce those rules with decentralized Timeboost.
+
+For Arbitrum chains, the sequencer’s sole job is to take arriving, valid transactions from users, place them into an order dictated by the transaction ordering policy, and then publish the final sequence to a real-time feed and in compressed batches to the chain’s data availability layer. The current transaction ordering policy is **F**irst **C**ome **F**irst **S**erve (FCFS) and Timeboost is a modified FCFS ordering policy.
+
+Timeboost is implemented using three separate components that work together: 
+- **A special “express lane”** which allows valid transactions to be sequenced as soon as the sequencer receives them for a given round.
+- **An off-chain auction** for determining the controller of the express lane for a given round. This auction is managed by an <a data-quicklook-from='autonomous-auctioneer'>autonomous auctioneer</a>.
+- **An <a data-quicklook-from='auction-contract'>auction contract</a>**, deployed on the target chain, to serve as the canonical source of truth for the auction results and handling of auction proceeds.
+
+To start, the default duration of a round is 60 seconds. Transactions not in the express lane will be subject to a default 200 milisecond “speed bump” for their transaction to be sequenced, which means that non-express lane transactions may get delayed to the next block. It’s important to call out that the default Arbitrum block time will remain at 250 miliseconds (or lower for Arbitrum chains if desired). Let’s dive in to how each of these components work.
```

Comment:
Now that we changed all "Orbit chains" for "Arbitrum chains", the sentence `(or lower for Arbitrum chains if desired)` seems weird. Maybe here is a good place to use Orbit chains, or something different but referring to chains other than One and Nova.

Reply:
Thanks good catch